tools/vrt-selector/src/story-candidates.js

```
@@ -0,0 +1,214 @@
+function listSourceFiles(dir) {
+  return walkFiles(dir).filter((file) => /\.(ts|tsx|css)$/.test(file));
+}
+
+function getAffectedPackages(changedFiles) {
+  const affectedPackages = new Set();
+
+  for (const rawFile of changedFiles) {
+    const file = normalizePath(rawFile);
+
+    if (file === "package.json" || file === "yarn.lock" || file.startsWith("libs/")) {
+      VRT_PACKAGES.forEach((pkg) => affectedPackages.add(pkg));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)